app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivity.kt

```diff
@@ -75,7 +80,7 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
     }
 
     private fun getLocationPermission() {
-        var permissions = arrayOf(
+        val permissions = arrayOf(
```

Comment:
naming conventions of the the variable - https://kotlinlang.org/docs/coding-conventions.html#property-names